app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml

```diff
@@ -17,9 +18,10 @@
       android:drawablePadding="8dp"
       android:gravity="center_vertical"
       android:text="@string/menu_options"
-      android:textColor="@color/color_def_black"
+      android:textColor="@color/component_color_shared_primary_dark_text_color"
       android:textSize="20sp"
-      app:drawableStartCompat="@drawable/ic_options_icon_grey_24dp" />
+      app:drawableStartCompat="@drawable/ic_options_icon_grey_24dp"
+      android:drawableTint="@color/component_color_shared_multipane_icon_color" />
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Done.